content/applications/sales/subscriptions/reports.rst

```diff
@@ -2,119 +2,374 @@
+- :guilabel:`Download xlsx`: when clicked the pivot data table is download as an .xlsx file.
+
+List view
+~~~~~~~~~
+
+With the list view selected, the subscription metrics being analyzed are displayed in a simple list,
+which can be fully customized by using any of the available filters in the drop-down filter menu
+(accessible via the :guilabel:`▼ (downward facing arrow)` icon to the right of the
+:guilabel:`Search...` bar).
+
+.. note::
+   With list view selected, the :guilabel:`Measures` drop-down menu and :guilabel:`Insert in
+   Spreadsheet` button are *not* available.
+
+Measures
+--------
+
+Each reporting page has its own metric-specific :guilabel:`Measures` drop-down menu of data-related
+options to choose from, located in the upper-left corner, above the visual representation of
+metrics.
+
+.. image:: reports/subscriptions-measures-drop-down.png
   :align: center
-  :alt: Difference between MRR and ARR in Odoo Subscriptions
+  :alt: The standard measures drop-down menu in the Odoo Subscriptions application.
+
+When clicked, a series of selectable measures become available, via a drop-down menu. When any of
+the options are selected from a :guilabel:`Measures` drop-down menu, the chosen metrics related
+to that specific measure appear on the reporting page.
+
+.. note::
+  For more information on the different measures that can be utilized on each reporting page, refer
+  to the specific reporting page breakdowns found below in this documentation.
+
+Insert in spreadsheet
+---------------------
+
+Beside the :guilabel:`Measures` drop-down menu, there is an :guilabel:`Insert in Spreadsheet`
+button.
+
+When clicked, the ability to add the configured data currently being showcased on the reporting page
+into a new or pre-existing spreadsheet (or dashboard) becomes available, via a pop-up window.
+
+.. image:: reports/subscriptions-analysis-spreadsheet-popup.png
+  :align: center
+  :alt: The spreadsheet pop-up window of the Subscriptions Analysis page.
+
+Select the desired option from this pop-up window, then click :guilabel:`Confirm`.
 
-- **Non-Recurring Revenue (NRR)**: NRR shows the revenue earned for everything else than
-  subscription-based products or services. This includes gains of a rare or unique nature that are
-  unlikely to occur in the ordinary course of businesses.
+Reporting pages
+===============
 
-- **Customer Retention**: Practices to engage existing customers to continue buying products or
-  services from your business. Customer retention can be a challenge, because you must prove you
-  are worthy of your customers' trust.
+In the Odoo *Subscriptions* application, there are four different reporting pages available.
+
+To access, analyze, and customize various reports related to subscriptions, navigate to
+:menuselection:`Subscriptions app`, and click the :guilabel:`Reporting` drop-down menu in the
+header.
+
+Clicking :guilabel:`Reporting` in the header menu of the *Subscriptions* application, reveals a
```

Comment:
I don't think this needs to be repeated from the previous paragraph. It's pretty self-explanatory that we're in the Subscriptions app for this doc
```suggestion
Clicking :guilabel:`Reporting` in the header menu reveals a
```